modules/javafx.base/src/main/java/javafx/beans/property/adapter/JavaBeanBooleanPropertyBuilder.java

```diff
@@ -62,8 +62,9 @@
     private final JavaBeanPropertyBuilderHelper helper = new JavaBeanPropertyBuilderHelper();
 
     /**
-     * <b>Do not use this constructor.</b> It will be deprecated in the next version. Use {@link #create()} instead.
+     * This constructor was exposed erroneously and will be removed in the next version. Use {@link #create()} instead.
      */
+    @Deprecated(since = "14", forRemoval = true)
```

Comment:
Minor: I checked other places in JavaFX and JDK, and they consistently omit the spaces surrounding the `=`.

Reply:
For both `since` and `forRemoval`?

Reply:
Yes. The pattern used consistently (in all cases that I could find) would be:

```
    @Deprecated(since="14", forRemoval=true)
```